src/main/java/frc/robot/subsystems/Underglow.java

```diff
@@ -0,0 +1,106 @@
+/* ==================================================
+ * Authors: Shane Pinto
+ *
+ * --------------------------------------------------
+ * Description: Code for LED Light strips underneath
+ * robot. Communicates via Serial using the RS-232
+ * port on the Roborio.
+ * 
+ * RGB Controller Repository: https://github.com/team4015/RGB-Controller
+ * ================================================== */
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.SerialPort;
+import edu.wpi.first.wpilibj.SerialPort.Port;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.Timer;
+
+public class Underglow extends SubsystemBase
+{
+  SerialPort uart;
+  
+  final int HEADER_BYTE = 0xFF;
+  final int BAUD_RATE = 9600;
+  final double TIMEOUT = 0.1; // 100 ms
+  final double TRANSMIT_DELAY = 0.01; // 10 ms
+
+  public Underglow()
+  {
+    uart = new SerialPort(BAUD_RATE, Port.kOnboard);
+
+    uart.setTimeout(TIMEOUT);
+  }
+
+  public void transmitByte(int value)
+  {
+    byte [] buffer = {(byte) value};
+    uart.write(buffer, buffer.length);
+    uart.flush();
+    Timer.delay(TRANSMIT_DELAY);
```

Comment:
Will this delay just the command that is calling it or the whole robot? It says in the documentation that it will pause the thread that it is in. Does a command count as a thread? If it doesn't you might want to create a thread that calls this method instead of calling it directly in `transmit()`, because pausing the whole robot will mess up the enoders and the auto align.

Reply:
I will try to eliminate the need for this delay by changing the PIC firmware.

Reply:
Was not able to eliminate the delay via firmware.  Maybe we can use `timer.start()` instead.

Reply:
`timer.start()` should allow us to wait on this thread using `while` instead of sleeping.  Hopefully that doesn't mess up the rest of the robot stuff (I don't think it will because we use timers elsewhere without any issues).  We shouldn't be setting the colour over and over anyways, so I think this delay will be neligible.

Reply:
Updated this to be an async timer so the delay will happen in the background.